app/database/models/__init__.py

```
@@ -1,7 +1,7 @@
 # Optional: just for clarity, not mandatory
-from .user import User
-from .chat import Chat
-from .message import Message
-from .prompt import Prompt
-from .code import Code
-from .video import Video
+from .chat import Chat as Chat
```

Flake8 (7.2.0)

[error] '.chat.Chat' imported but unused

(F401)